Sources/Extensions/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -40,39 +40,50 @@ public extension Optional {
         return wrapped
     }
 
-	/// SwifterSwift: Runs a block to Wrapped if not nil
-	///
-	///		let foo: String? = nil
-	///		foo.run { unwrappedFoo in
-	///			// block will never run sice foo is nill
-	///			print(unwrappedFoo)
-	///		}
-	///
-	///		let bar: String? = "bar"
-	///		bar.run { unwrappedBar in
-	///			// block will run sice bar is not nill
-	///			print(unwrappedBar) -> "bar"
-	///		}
-	///
-	/// - Parameter block: a block to run if self is not nil.
-	public func run(_ block: (Wrapped) -> Void) {
-		// http://www.russbishop.net/improving-optionals
-		_ = self.map(block)
-	}
+    /// SwifterSwift: Runs a block to Wrapped if not nil
+    ///
+    ///		let foo: String? = nil
+    ///		foo.run { unwrappedFoo in
+    ///			// block will never run sice foo is nill
+    ///			print(unwrappedFoo)
+    ///		}
+    ///
+    ///		let bar: String? = "bar"
+    ///		bar.run { unwrappedBar in
+    ///			// block will run sice bar is not nill
+    ///			print(unwrappedBar) -> "bar"
+    ///		}
+    ///
+    /// - Parameter block: a block to run if self is not nil.
+    public func run(_ block: (Wrapped) -> Void) {
+        // http://www.russbishop.net/improving-optionals
+        _ = self.map(block)
+    }
+
+    /// SwifterSwift: Assign an optional value to a variable only if the value is not nil.
+    ///
+    ///     let someParameter: String? = nil
+    ///     let parameters = [String:Any]() //Some parameters to be attached to a GET request
+    ///     parameters[someKey] ??= someParameter //It won't be added to the parameters dict
+    ///
+    /// - Parameters:
+    ///   - lhs: Any?
+    ///   - rhs: Any?
+    public static func ??= (lhs: inout Optional, rhs: Optional) {
+        guard let rhs = rhs else { return }
+        lhs = rhs
+    }
+
+}
+
+// MARK: - Methods (RandomAccessCollection)
```

Comment:
RandomAccessCollection comment is right?

Reply:
No, my bad, fixing it now 😅